### ISSUE TITLE
%essentials_safe_online% throws exception

### ISSUE THREAD

**Issue by Lusteroth** [EssentialsX/Essentials#3660]
### Information

**Full output of `/ess version`**:

```
[21:22:35 INFO]: Server version: 1.16.2-R0.1-SNAPSHOT git-Paper-186 (MC: 1.16.2)
[21:22:35 INFO]: EssentialsX version: 2.18.1.0
[21:22:35 INFO]: EssentialsXSpawn version: 2.18.1.0
[21:22:35 INFO]: EssentialsXChat version: 2.18.1.0
[21:22:35 INFO]: PlaceholderAPI version: 2.10.9
[21:22:35 INFO]: Vault is not installed. Chat and permissions may not work.
```

**Server log**: https://gist.github.com/Lusteroth/9b8dd0612913c5996d0f3d166c1e307d

**EssentialsX config**: https://gist.github.com/Lusteroth/7783c9ed19c2073d4395960462cf6e85

### Details

**Description**  
```%essentials_safe_online%``` isn't working in version 2.18.1.0. (In 2.18.0 it's fine)

**Steps to reproduce**  
Create a server -> upload/install EssentialsX & PlaceholderAPI -> try to use the placeholder ```%essentials_safe_online%```

**Expected behavior**  
```%essentials_safe_online%``` should not produce an error / should work.

**Screenshots**  
not necessary

**Comment by Chew** [comment]
1) We don't manage the Essentials expansion.
2) Duplicate of PlaceholderAPI/Essentials-Expansion#26

**Comment by pop4959** [comment]
2.18.1.0 is not the latest version. Please update to the latest dev build and try again. I think this may have been fixed already today.

**Comment by Lusteroth** [comment]
You're right - the issue was fixed today 👍 (1382a05dc85f5f22f6bb589307a6bee9f9ebc0cf)
